RepClient/src/app/shared/component/add-remove-item/add-remove-item.component.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { IonicModule } from '@ionic/angular';
+import { MockStore, provideMockStore } from '@ngrx/store/testing';
+import * as fromOrder from '../../../order/store/order.reducer';
+import * as fromApp from '../../../reducers/index';
+import { SharedModule } from '../../shared.module';
+
+import { AddRemoveItemComponent } from './add-remove-item.component';
+
+describe('AddRemoveItemComponent', () => {
+  let component: AddRemoveItemComponent;
+  let fixture: ComponentFixture<AddRemoveItemComponent>;
+  let mockStore: MockStore<fromApp.State>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ AddRemoveItemComponent ],
+      imports: [
+        IonicModule.forRoot(),
+        SharedModule
+      ],
+      providers: [
+        provideMockStore()
+      ]
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(AddRemoveItemComponent);
+    component = fixture.componentInstance;
+    mockStore = TestBed.inject(MockStore);
+    const mockItemsSelector = mockStore.overrideSelector(
+      fromOrder.selectItems,
+      []
+    );
+
+    fixture.detectChanges();
+  }));
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
This test alone is not enough

Reply:
Tests where deemed not appropriate for this pull request and would be added as a separate pull request. However that was put on hold for now to aid with the Admin front end as it is more pressing to have it finished.